src/test/java/com/solvd/apptest/AndroidTest.java

```diff
@@ -0,0 +1,58 @@
+package com.solvd.apptest;
+
+import com.solvd.apptest.pages.android.LoginError;
+import com.solvd.apptest.pages.base.CatalogPageBase;
+import com.solvd.apptest.pages.base.LoginPageBase;
+import com.zebrunner.carina.core.IAbstractTest;
+import com.zebrunner.carina.utils.R;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class AndroidTest implements IAbstractTest {
+    @Test
+    public void validLoginTest() {
+        CatalogPageBase catalogPage = initPage(getDriver(), CatalogPageBase.class);
+        catalogPage.assertPageOpened(3000);
```

Comment:
I think Carina default time here is enough so you can get rid off of 3000 value.

Reply:
I think if check fails `assertPageOpened` tries again for 20 seconds so setting it to 3000ms can make it faster